Druid/DRAbstractCompilerBuilder.class.st

```diff
@@ -136,6 +138,26 @@ DRAbstractCompilerBuilder >> primitiveTable [
 	^ self subclassResponsibility
 ]
 
+{ #category : #accessing }
+DRAbstractCompilerBuilder >> selectBytecodes: selectBlock [
+	" Answer a <Collection> of <CompiledMethod> with bytecodes matching conditions in selectBlock "
+
+	self shouldBeImplemented
+]
+
+{ #category : #accessing }
+DRAbstractCompilerBuilder >> selectPrimitives: selectBlock [
+	" Answer a <Collection> of <CompiledMethod> representing primitive methods and matching conditions in selectBlock "
+	| primitiveCandidates |
+
+	primitiveCandidates := self
+		matchMethod: #beginsWith:;
```

Comment:
This is strange.
The input of the compiler is the primitive table from the interpreter.
There is no need to scan the list of methods, right?

Otherwise we could create a pair interpreter-compiler that is inconsistent